www/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "paragon-pattern-library-documentation",
+  "description": "Paragon Pattern Library Documentation",
+  "version": "1.0.0",
+  "dependencies": {
+    "@mdx-js/mdx": "^1.0.10",
+    "@mdx-js/react": "^1.0.6",
+    "gatsby": "^2.3.24",
+    "gatsby-mdx": "^0.6.2",
+    "gatsby-plugin-manifest": "^2.0.29",
+    "gatsby-plugin-react-helmet": "^3.0.12",
+    "gatsby-plugin-sass": "^2.0.11",
+    "gatsby-source-filesystem": "^2.0.29",
+    "node-sass": "^4.12.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-helmet": "^5.2.0"
+  },
+  "keywords": [
+    "paragon",
+    "pattern library",
+    "design",
+    "react",
+    "component library"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
```

Comment:
I'd recommend we add a `-p <some more obscure number that nothing in devstack uses>` flag now. Will help folks who live with devstack up leverage this sooner

Reply:
What's your favorite number?

Reply:
Actually it looks like Gatsby will ask if you'd like to run on a different port. It defaults to 8000 and works up from there. I think that should work well enough for now.